frontend/app/router.js

```diff
@@ -0,0 +1,16 @@
+import Ember from 'ember';
+import config from './config/environment';
+
+const Router = Ember.Router.extend({
+  location: config.locationType,
+  rootURL: config.rootURL
+});
+
+Router.map(function() {
+  this.route('signIn');
```

Comment:
Files are already properly name. Only the router.js file and anywhere you have used `link-to`. A simple find and replace should do.